app/assets/stylesheets/fuel/_post-list.scss

```diff
@@ -30,13 +32,17 @@ $post-date-color:   $grey-dark;
     text-transform: uppercase;
   }
 
-  .post-item--title{
+  .post-item--title, .author-item--name{
```

Comment:
Each selector in a comma sequence should be on its own line<br>Opening curly brace `{` should be preceded by one space